apps/web/src/app/(signing)/sign/[token]/page.tsx

```diff
@@ -55,6 +55,10 @@ export default async function SigningPage({ params: { token } }: SigningPageProp
     redirect(`/sign/${token}/complete`);
   }
 
+  if (document.deletedAt !== null) {
+    redirect(`/sign/${token}/deleted`);
+  }
```

Comment:
Nice, I like that this occurs after the completion check so for normal recipients with a completed document there's nothing more they need to do